### ISSUE TITLE
Undefined variable 'sx' in npm package

### ISSUE THREAD

**Issue by gurupras** [Foliotek/Croppie#629]
As far as I can tell, the npm package has an error.
Looking at [unpkg.com](https://unpkg.com/croppie), if you search for `sx =`, you'll find
```
sx = left;
sy = top;
```
But these variables don't have a prefixed `var` or `let` and thus they cause errors while trying to use them. The github repo doesn't seem to have this problem as you can see:
https://github.com/Foliotek/Croppie/blob/680db73a21c43fbaef26bd3d4fbb2dbdb63c2341/croppie.js#L1183-L1184

## Expected Behavior
Croppie works when using the npm package

## Actual Behavior
Crashes when trying to crop due to `ReferenceError`
```
ReferenceError: sx is not defined
    at oe.G (croppie.js:1183)
    at oe.Z (croppie.js:1262)
    at croppie.js:1444
    at new Promise (<anonymous>)
    at oe.ee (croppie.js:1436)
    at oe.result (croppie.js:1608)
```

**Comment by jofftiquez** [comment]
Same issue here. Am using `1.6.3`

**Comment by alsmpd** [comment]
@gurupras this is fixed in v2.6.4 (https://unpkg.com/croppie@2.6.4/croppie.js)

Thanks.
